upgrade/upgrade-provider.go

```diff
@@ -220,6 +220,15 @@ func UpgradeProvider(ctx Context, repoOrg, repoName string) error {
 			"go", "get", "github.com/pulumi/pulumi-terraform-bridge/v3@"+targetBridgeVersion)).
 			In(repo.providerDir()))
 	}
+	if ctx.UpgradeSdkVersion {
+		steps = append(steps,
```

Comment:
I think its nice if users see a clear correspondence between their inputs and outputs:
```suggestion
		steps = append(steps, step.Combined("Upgrade Pulumi SDK",
```